stages/org.osbuild.dnf.config

```diff
@@ -60,13 +78,43 @@ def configure_variable(tree, name, value):
     with open(f"{tree}{vars_directory}/{name}", "w") as f:
         f.write(value + "\n")
 
+def make_value(value):
+    val = str(value)
+    return val
+
```

Comment:
Super nit: two newlines between global functions in Python.

Reply:
those are the best ones